### DISCUSSION TITLE
Visual Studio 17.5.0-pre.2.0 wants to install additional components

### DISCUSSION THREAD

**Discussion by MarkStega** [unoplatform/uno#10850] [General]
Running "uno-check --pre" gives the 'thumbs up' everything looks great.

After creating a new Uno platform project Visual Studio prompts for additional components:
![devenv_2022-12-28_13-55-44](https://user-images.githubusercontent.com/634561/209859556-5e0b1413-fb59-4937-9307-ae10f15b78c1.png)

The list presented by the VS installer is:
![setup_2022-12-28_13-47-11](https://user-images.githubusercontent.com/634561/209859631-69d7b7bc-0c88-448d-8a12-3404b62ed6f7.png)

Can I just presume that the VS message should just be ignored?

**Comment by jeromelaban** [comment]
You can ignore those components at this time. Some of them are outdated and are [included in this file](https://github.com/unoplatform/uno/blob/master/src/SolutionTemplate/UnoSolutionTemplate.Wizard.2022/.vsconfig.vs17). We're tracking it to [make it conditional to the options used](https://github.com/unoplatform/uno/issues/9357) when creating the template.
